src/__stories__/cover-hero-story.tsx

```diff
@@ -0,0 +1,220 @@
+}) => {
+    const backgroundProps =
+        background === 'image'
+            ? {
+                  backgroundImage: usingVrImg,
+              }
+            : background === 'video'
+            ? {
+                  backgroundVideo: beachVideo,
+                  poster: beachImg,
+              }
+            : {
+                  background: backgroundColorFromSkin || backgroundColorCustom,
+                  variant,
+              };
+
+    const button = actions.includes('button') ? <ButtonPrimary fake>Action</ButtonPrimary> : undefined;
+    const buttonLink = actions.includes('link') ? <ButtonLink href="#">Link</ButtonLink> : undefined;
+    const secondaryButton = actions.includes('secondaryButton') ? (
+        <ButtonSecondary fake>Secondary</ButtonSecondary>
+    ) : undefined;
+
+    return (
+        <CoverHero
+            dataAttributes={{testid: 'cover-hero'}}
+            headline={headline ? <Tag type={headlineType}>{headline}</Tag> : undefined}
+            pretitle={pretitle}
+            title={title}
+            description={description}
+            extra={withExtra ? <Placeholder /> : undefined}
+            sideExtra={withSideExtra ? <Placeholder /> : undefined}
+            button={button}
+            secondaryButton={secondaryButton}
+            buttonLink={buttonLink}
+            minHeight={minHeight}
+            aspectRatio={aspectRatio}
+            centered={centered}
+            noPaddingY={noPaddingY}
+            {...backgroundProps}
+        />
+    );
+};
+
+Default.storyName = 'CoverHero';
+
+Default.args = {
+    background: 'image',
+    backgroundColorCustom: '',
+    backgroundColorFromSkin: '',
+    variant: 'default',
+    headlineType: 'promo',
+    headline: 'Hero',
+    pretitle: 'Pretitle',
+    title: 'Title',
+    description: 'This is a long description with a long text to see how this works',
+    withExtra: false,
+    withSideExtra: false,
+    actions: 'button and link',
+    minHeight: undefined,
+    aspectRatio: 'auto',
+    centered: false,
+    noPaddingY: false,
+};
+
+Default.argTypes = {
+    headlineType: {
+        options: ['promo', 'active', 'inactive', 'success', 'warning', 'error'],
+        control: {type: 'select'},
+    },
+    background: {
+        options: ['image', 'video', 'color from skin', 'custom color'],
+        control: {type: 'select'},
+    },
+    backgroundColorCustom: {
+        control: {type: 'color'},
+        if: {arg: 'background', eq: 'custom color'},
+    },
+    backgroundColorFromSkin: {
+        control: {type: 'select'},
+        options: {'none (determined by variant)': '', ...skinVars.colors},
+        if: {arg: 'background', eq: 'color from skin'},
+    },
+    variant: {
+        options: ['default', 'inverse', 'alternative'],
+        control: {type: 'select'},
+        // this control should only be visible when background is set to 'color from skin' or 'custom color'
+        // if: {arg: 'background', eq: 'color'},
```

Comment:
?

Reply:
This is a missing storybook feature. I can configure the variant control to appear when background is `'color from skin'`: `if: {arg: 'background', eq: 'color from skin'}` but I can't configure it to appear when background is `'color from skin'` **or** `'custom color'`

Reply:
Yes, it's a very annoying thing from Storybook, but why leaving this commented line? I'd remove it

Reply:
just to document the intention, but I can remove it

Reply:
I've rewritten the comment with a better explanation